.schema/devbox-plugin.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://json-schema.org/draft-04/schema#",
+  "$id": "https://github.com/jetpack-io/devbox",
```

Comment:
This should be "https://github.com/jetpack-io/devbox-plugin" to differentiate between devbox schema and devbox plugin schema

Reply:
Do you have ideas on how we should change it? I'd like to keep them in the same repo